assets/css/main.css

```diff
@@ -1675,9 +1675,25 @@ input[type=radio] + label:before {
 }
 .image.fit img {
   width: 100%;
+  height: 200px;
```

Comment:
why is height static but width is not? is it because you want it to be rectangle rather than square in most screens?

Reply:
I updated the heights to instead have a max-height but calculate height based on the calculated width. I like the square look now